pyproject.toml

```diff
@@ -100,8 +100,9 @@ export = [
     "onnx>=1.12.0", # ONNX export
     "coremltools>=7.0; platform_system != 'Windows' and python_version <= '3.11'", # CoreML supported on macOS and Linux
     "openvino>=2024.0.0", # OpenVINO export
-    "tensorflow<=2.13.1; python_version <= '3.11'", # TF bug https://github.com/ultralytics/ultralytics/issues/5161
-    "tensorflowjs>=3.9.0; python_version <= '3.11'", # TF.js export, automatically installs tensorflow
+    "tensorflow>=2.0.0", # TF bug https://github.com/ultralytics/ultralytics/issues/5161
+    "tensorflowjs>=3.9.0", # TF.js export, automatically installs tensorflow
+    "keras",  # not installed auotomatically by tensorflow>=2.16
     "flatbuffers>=23.5.26,<100; platform_machine == 'aarch64'", # update old 'flatbuffers' included inside tensorflow package
```

Comment:
I think we might be able to remove this line now that we are using the latest TF?

Reply:
@glenn-jocher we cannot keep just `flatbuffers` because of this:

![image](https://github.com/ultralytics/ultralytics/assets/20147381/d5a28f9a-60a5-4194-a39c-a6bb36be4a94)

Reply:
But not sure if the new versions of TF now include newer versions of flatbuffers? Comment says we need to update because older TF has older flatbuffers.